gridt/src/app/settings/change-bio/change-bio.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
```

Comment:
Question: why is there an entire folder and module for a single component? Can this not be resolved by simply adding the component in the settings folder?